### PR TITLE
Setup find wallet page for translation

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Setup find wallet page for translation

### CODE REVIEW

**Comment by minimalsm** [comment]
@corwintines our tooltip descriptions aren't showing properly 🤔 the key exists and is correct...

Screenshot 2023-01-12 at 23 12 19

**Comment by corwintines** [comment]
@minimalsm this should be good to review again. Nice catch, sorry for missing that.

File: src/intl/en/page-wallets-find-wallet.json [hunk 1]
```diff
@@ -1,142 +1,92 @@
...
+  "page-find-wallet-nfts-title": "NFTs",
+  "page-find-wallet-nfts-desc": "You are someone that is all about NFTs and want a wallet with NFT support",
+  "page-find-wallet-hodler-title": "Hodler",
+  "page-find-wallet-hodler-desc": "You are someone that has tokens and don’t want to touch them",
+  "page-find-wallet-finance-title": "Finance",
+  "page-find-wallet-finance-desc": "You are someone that uses DeFi and want a wallet that allows you to connect to DeFi applications",
+  "page-find-wallet-developer-title": "Developer",
+  "page-find-wallet-developer-desc": "You are developer and need a wallet to help develop and test dapps",
+  "page-find-wallet-persona-desc": "Choose the profile that matches your type of user and filter the wallet list",
+  "page-find-wallet-filters": "FILTERS",
```
Comment (wackerow) [inline src/intl/en/page-wallets-find-wallet.json:78]: Would suggest adding this in regular casing, and use styling to make it uppercase to limit issues via translation.

Comment (corwintines) [reply src/intl/en/page-wallets-find-wallet.json:78]:
```suggestion
  "page-find-wallet-filters": "Filters",
```

**Review by wackerow — APPROVED** [review]

**Pull request merged**